File: src/python/fibonacci_float.py
import os
import time

def main():
    try:
        count = int(os.environ.get("COUNT", 1475))
    except (ValueError, TypeError):
        count = 1475

    a, b = 0.0, 1.0
    start = time.perf_counter()
    for _ in range(count):
        a, b = b, a + b
    end = time.perf_counter()
    
    print(f"Result(F_{count}): {a:.10e}")
    print(f"Time: {(end - start) * 1000:.3f} ms")

if __name__ == "__main__":
    main()

File: src/go/fibonacci_float.go
package main

import (
	"fmt"
	"os"
	"strconv"
	"time"
)

func main() {
	n := 1475
	if countStr := os.Getenv("COUNT"); countStr != "" {
		if val, err := strconv.Atoi(countStr); err == nil {
			n = val
		}
	}

	var a, b float64
	
	start := time.Now()
	for k := 0; k < 200000; k++ {
		a = 0.0
		b = 1.0
		for i := 0; i < n; i++ {
			temp := a
			a = b
			b = temp + b
		}
	}
	elapsed := time.Since(start)

	fmt.Printf("Result: %e\n", a)
	fmt.Printf("Time: %.3f ms\n", float64(elapsed.Milliseconds()))
}
